tests/math/test_matrix_manipulation.py

```diff
@@ -487,7 +488,8 @@ def test_expand_matrix_usage_in_operator_class(self, tol):
         class DummyOp(qml.operation.Operator):
             num_wires = 2
 
-            def compute_matrix(*params, **hyperparams):
+            @staticmethod
+            def compute_matrix():
```

Comment:
I'm surprised that there isn't a pylint error here since the interface for `Operator.compute_matrix` is being overridden.

Reply:
I think it is because the original method has a flexible number of arguments/kwarguments.